lib_nbgl/doc/nbgl_use_case.dox

```diff
@@ -653,122 +326,107 @@ static nbgl_layoutTagValue_t* getPair(uint8_t index) {
   return &pair;
 }
 
-void reviewContinue(void) {
-  static nbgl_pageInfoLongPress_t infoLongPress = {
-    .icon = &myAppIcon,
-    .text = "Confirm transaction\nMyAppCoin send",
-    .longPressText = "Hold to send"
-  };
-
+void staticReview(void) {
   // static review, providing the whole list of pairs
-  nbgl_useCaseStaticReview(&pairList, &infoLongPress, "Reject transaction", reviewChoice);
+  nbgl_useCaseReview(TYPE_TRANSACTION, // type of operation
+                      &pairList,       // list of tag/value pairs
+                      coinIcon,        // icon of the coin
+                      "Review transaction\nto send coin", // title of the first page
+                      NULL,                               // sub-title of the first page
+                      "Sign transaction to\nsend coin?",  // title of the last page
+                      onReviewResult);                    // callback on result of the review
 }
 @endcode
 
-@subsection use_case_static_review_light Light Static Review Use Case
-
-\image{inline} html UseCase-ReviewLight.png "caption" height=300
+@subsection use_case_review_light Light Static Review Use Case
 
 In some cases, the developer may want to display a review but with a less intense confirmation than a long-press button. A simple button is used in this
 case.
 
-The API to initiate the display of the series of pages is @ref nbgl_useCaseStaticReviewLight(), providing:
+The API to initiate the display of the series of pages is @ref nbgl_useCaseReviewLight(), providing:
 
 - the list of tag/value pairs (or a callback to get them one by one)
-- the texts to use in footer and in last page (icon, title, confirmation button text)
-- a callback called when the black button on last page or the footer to reject is used. The callback's param is true for confirmation, false for rejection.
+- the texts/icon to use in presentation page and in last page
+- a callback called when the long press button on last page or reject confirmation is used. The callback's param is *true* for confirmation, *false* for rejection.
+
+@subsection use_case_review_streaming Streaming review Use Case
+
+\image{inline} html UseCase-Streaming.png "caption" height=300
+
+In some cases, the application cannot know all tag/value pairs of a transaction when the review is started.
+
+In this case, what we call a "streaming" review can be used. The pages to display for each "stream are computed automatically and pages can be navigated forward
+and backward (only within a "stream" for backward).
```

Comment:
Maybe I am wrong, but my understanding is that with streming, where the pairs are not known in advance, the back button is not usable, because the information is not stored, but displayed as they come?

Reply:
If you provide 10 pairs in the continue() function, you can navigate back and forward in these pairs...